ravenframework/Optimizers/GeneticAlgorithm.py

```diff
@@ -524,6 +524,25 @@ def _useRealization(self, info, rlz):
                                                variables=list(self.toBeSampled))
 
       # 4 @ n: repair/replacement
+      # # Make sure no children are exactly similar to parents
```

Comment:
Please remove the commented block. Notice that we will revisit this when addressing adaptive crossover, and mutator.